services/motion/builtin/builtin.go

```diff
@@ -150,6 +150,10 @@ func (ms *builtIn) Move(
 	if err != nil {
 		return false, err
 	}
+	if ctx.Err() != nil {
+		ms.logger.Infof("successfully canceled motion service Move")
+		return true, ctx.Err()
+	}
```

Comment:
TODO: in the check for the error of `resources[name].GoToInputs(ctx, inputs)` ensure all other components have stopped moving.